torchvision/transforms/transforms.py

```diff
@@ -321,7 +321,14 @@ def __repr__(self):
 
 
 class RandomHorizontalFlip(object):
-    """Horizontally flip the given PIL Image randomly with a probability of 0.5."""
+    """Horizontally flip the given PIL Image randomly with a probability.
```

Comment:
Maybe you could say something like
```
Horizontally flip the given PIL Image randomly with a given probability
```
?
Same thing for the Vertical flip, what do you think?

Reply:
I agree